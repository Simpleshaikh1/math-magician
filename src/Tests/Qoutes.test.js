import { render } from '@testing-library/react';
import Quotes from '../routes/displayQoute';

test('renders Quotes correctly', () => {
  const { container } = render(<Quotes />);
  expect(container).toMatchSnapshot();
});
