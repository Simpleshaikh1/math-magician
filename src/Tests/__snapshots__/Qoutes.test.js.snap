// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Quotes correctly 1`] = `
<div>
  <p>
    Loading...
  </p>
</div>
`;
